### PR TITLE
Add Rails 6 support

### PR DESCRIPTION
**Pull request by JevgenijusB**
@zverseckas

### CODE REVIEW

File: chronopost.gemspec [hunk 1]
```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |spec|
   spec.summary = 'Chronopost client'
   spec.description = 'An interface for the Chronopost Web Service'
   spec.homepage = 'https://github.com/vinted/chronopost'
-  spec.required_ruby_version = '~> 2.0'
+  spec.required_ruby_version = '~> 2.6.4'
```
Comment (zverseckas) [inline chronopost.gemspec]: This is not good. We should keep the lowest compatible version of ruby. :)

File: chronopost.gemspec [hunk 2]
```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_runtime_dependency 'activesupport', '~> 5.2', '>= 5.1.2'
+  spec.add_runtime_dependency 'activesupport', '~> 6.0.0', '>= 6.0.0'
```
Comment (zverseckas) [inline chronopost.gemspec]: I think it should be OK to only leave the `~> 6.0` part.

File: .ruby-version [hunk 3]
```diff
@@ -1 +1 @@
-2.6.3
+2.6.4
```
Comment (evaldasg) [inline .ruby-version]: Is this version available on server?

Comment (JevgenijusB) [reply .ruby-version]: This one is for development purposes only

File: chronopost.gemspec [hunk 4]
```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_runtime_dependency 'activesupport', '~> 5.2', '>= 5.1.2'
+  spec.add_runtime_dependency 'activesupport', '~> 6.0', '>= 6.0.0'
```
Comment (zverseckas) [inline chronopost.gemspec]: The `>=` part is covered by `~> 6.0`.

File: .ruby-version (hunk 3 above)
Comment (evaldasg) [reply .ruby-version]: no plan to deploy live?

Comment (JevgenijusB) [reply .ruby-version]: As I'm aware, ruby version for server is defined in chronopost.gemspec

Comment (zverseckas) [reply .ruby-version]: Travis CI uses a version specified in `.travis.yml`. When a gem is installed on some system it uses ruby and dependency versions specified in gemspec. :)

**Pull request merged**